corehq/tests/noseplugins/timing.py

```diff
@@ -114,3 +121,16 @@ def end_event(name, context):
     event. Requires the `TimingPlugin` must to be enabled.
     """
     PLUGIN_INSTANCE.end_event(name, context)
+
+
+def _get_expected_slow_seconds(test_context):
+    # class is tagged for class based test
+    # For function based test, the function is tagged
+    slow_seconds = getattr(test_context, 'slow', None)
```

Comment:
If the class is tagged, does that time apply to each test? For example, if a class having three test methods is tagged `slow=15` then each test gets a timeout of 15s for a total of 45s overall? Or is it 15s total for the whole class?

Reply:
It applies to each class